src/screens/SignInV2/elements/BottomOverlay/index.js

```diff
@@ -60,7 +61,7 @@ const BottomOverlay = ({count, handleLogin, index, isLogin, title, onNextSlide,
   return (
     <View style={bottomOverlayStyles.container}>
       <Text style={bottomOverlayStyles.title}>{title}</Text>
-      <View>{textSvg}</View>
+      <Text style={bottomOverlayStyles.contentText}>{text}</Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This line has been changed to render the `text` prop as a `<Text>` component instead of rendering `textSvg` as a child component. This is consistent with the change made on line 17.

```diff
- <View>{textSvg}</View>
+ <Text style={bottomOverlayStyles.contentText}>{text}</Text>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->